README.md

```diff
@@ -1,44 +1,44 @@
 # NL Design Skeleton Application
 
-The NL Design Skeleton application is design for rapid application testing and prototype development on the NL Design System. It does so by providing a basic skeleton application with full NL Design system functionality that an developer can easily extend view locally and deploy to an online environment for demonstration purposes. the main benefits of this are
+The NL Design Skeleton application is designed for rapid application testing and prototype development on the NL Design System. It provides a basic skeleton application with full NL Design system functionality that any developer can easily extend view locally and deploy to an online environment for demonstration purposes. The main benefit of this is:
 
-Development and (online) demonstration of prototypes without the need of an servers
-An ready out of the box basic  application that doesn't require configuration or setup and can be extended immediately.
+Development and (online) demonstration of prototypes without the need of a server.
+An out-of-the-box basic application that doesn't require configuration or setup and can be extended immediately.
 
-The application is available in two versions, a next.js version and an gatsby version. If you want to put your prototype online without setting up a server we recommend the gatsby version, since that can be deployed through gatsby pages.
+The application is available in two versions, a Next.js version, and a Gatsby version. We recommend the Gatsby version with deployment through Gatsby pages if you want to deploy your prototype without setting up a server.
 
 ## Getting started
 
-In order to set up your own project you will need an github account and be logged in, the simply click on “use this template” button. Tell github where you want to spin up your prototype and click “create repository from template”.
+To set up your own project, you will need a GitHub account and be logged in. Simply click on the "use this template" button. Tell GitHub where you want to spin up your prototype and click "create a repository from template".
 
+## Spinning up your local environment
 
-## Spinning up you  local environment
+You will need a git client, Node.js, and Yarn installed on your local machine for this part of the tutorial.
 
-For this part of the tutorial you will need and git client, node.js, and yarn on your  local machine
-
-In order to start developing locally you need to clone your new repository to your local machine. Then open your favorite CLI and simply navigate to the folder containing your repository and start up gatsby or next.js in development mode with the following commands.
+To develop locally, clone your new repository to your local machine. Open the terminal, navigate to the folder containing your repository, and start up Gatsby or Next.js in development mode with the following commands.
 
 **Nextjs**
 
-```cli
+"`cli
```

Comment:
Dit klopt nu niet helemaal met de instructies hierboven. Je moet namelijk deze commands niet in de root (`/`) van het project uitvoeren, maar in de pwa folder (`/pwa`).

Kunnen we denk ik het beste oplossen door vóór de install commands een extra command toe te voegen:

```
$ cd pwa
```

Reply:
(Mocht je onbekend zijn met het command, staat voor _change directory_)